miso-web/src/main/webapp/styles/style-full.css

```diff
@@ -698,11 +698,11 @@ div.checklist {
     font-weight: bold;
 }
 
-/*
+
 .widget_title input[type="text"] {
- float: right;
+    margin-left: 5px;

```

Comment:
Nitpicky note:
Eagle-eyed web devs will note that the Project and Run widget make space between the title and the search bar by using a literal space (eg. `Project <input>`).  Thus, the spacing between the title and search bar is a little over-spaced for the Project and Run widgets, but this will be fixed when they are refactored to use the utilities in `pane.js` and `tile.js`.

Reply:
If you're losing sleep over this, you could try using flex layout. https://css-tricks.com/almanac/properties/j/justify-content/